crates/aptos/src/config/mod.rs

```diff
@@ -112,6 +118,47 @@ impl CliCommand<GlobalConfig> for SetGlobalConfig {
     }
 }
 
+/// Show the private key for the given profile
```

Comment:
I was avoiding this, so people don't go copy their private key all over the place, but I guess it's inevitable.

Can we make one for Address then?

Reply:
> one for Address

This is already accessible via `show-profiles --profile` with a parser like `jq`, so I don't know if this is needed.

Private key, however, wasn't available, because it is explicitly hidden from `show-profiles` for security. I agree with the design decision, so I think the only command that is truly necessary is the new standalone to get private key

Not opposed to someone else adding it in a followup PR, of course, but I'd rather get this in first

Reply:
Oh right, yeah ignore that then.